### PR TITLE
Add Navigation Bar 

### PR DESCRIPTION
**Pull request by Jasminewwwu**

### CODE REVIEW

**Comment by nigef** [comment]
Thanks for getting this started @Jasminewwwu, good idea to add a nav bar!

A good idea is to populate the items dynamically on the front end, with javascript, to ensure the list always has the correct items.

We should also try to make it slightly more semantic, and bootstrap-y.

And there are a few more things I mentioned in the issue. 

I can help with that after our next meeting, if you like.

File: pyta_template.html [hunk 1]
```diff
@@ -14,7 +14,158 @@
     <link rel="stylesheet" href="$css$" $if(html5)$$else$$$type="text/css" $endif$/>
   $endfor$
 </head>
+
 <body>
+
+<ul id="nav">
+  <li class="dropdown">
+    <a href="index.html#Elif checker" class="dropbtn">Elif checker</a>
```
Comment (david-yz-liu) [inline pyta_template.html:22]: You don't need the link here, you can just use `href="#"` (applies to all sections).

File: pyta_template.html [hunk 2]
```diff
@@ -14,7 +14,158 @@
...
+    <a href="index.html#C0103">C0103 Invalid Name</a>
+    <a href="index.html#C0112">C0112 Empty Docstring</a>
+    <a href="index.html#C0113">C0113 Unneeded not</a>
+    <a href="index.html#C0121">C0121 Singleton comparison</a>
+    <a href="index.html#C0123">C0123 Unidiomatic type check</a>
+    <a href="index.html#E0101">E0101 Return in init</a>
+    <a href="index.html#E0102">E0102 Function redefined</a>
+    <a href="index.html#E0103">E0103 Not in loop</a>
+    <a href="index.html#E0104">E0104 Return outside function</a>
+    <a href="index.html#e0108-duplicate-argument-name-e0108 ">E0108 Duplicate argument name</a>
```
Comment (david-yz-liu) [inline pyta_template.html:100]: This change is not appropriate. The problem is not with the link, it is in how the header is generated (so please fix it elsewhere).

In general, any inconsistency is probably not correct.

**Comment by david-yz-liu** [comment]
@Jasminewwwu I will merge this branch into the `nav-bar` branch instead.

Please re-make the pull request, but choose the base (target) branch to be `nav-bar` instead of `master`

**Pull request closed without merging**